app/src/main/java/com/sakaimobile/development/sakaiclient20/ui/SakaiApplication.java

```diff
@@ -11,17 +12,20 @@
 import dagger.android.AndroidInjector;
 import dagger.android.DispatchingAndroidInjector;
 import dagger.android.HasActivityInjector;
+import dagger.android.support.HasSupportFragmentInjector;
 import io.fabric.sdk.android.Fabric;
 
 public class SakaiApplication extends Application
-        implements HasActivityInjector {
+        implements HasActivityInjector, HasSupportFragmentInjector {
 
     @Inject DispatchingAndroidInjector<Activity> activityInjector;
 
+    @Inject DispatchingAndroidInjector<Fragment> supportFragmentInjector;
+
     @Override
     public void onCreate() {
         super.onCreate();
-        Fabric.with(this, new Crashlytics());
+//        Fabric.with(this, new Crashlytics());
```

Comment:
Don't comment this out, there's a proper way to disable Crashlytics in debug mode that's in the assignments branch and will get added when I make a separate PR.